java-squid/src/main/java/org/sonar/java/model/VisitorsBridge.java

```diff
@@ -242,6 +250,37 @@ public String getFileKey() {
       return sourceFile.getKey();
     }
 
+    @CheckForNull
+    private RuleKey getRuleKey(JavaCheck check) {
```

Comment:
This is not related to this ticket, but doing this should probably lead to remove completely the ChecksBridge class and report directly from the context class. If you share the feeling can you please create the ticket ?